api/src/main/java/edu/cornell/mannlib/vitro/webapp/dynapi/components/N3Template.java

```diff
@@ -6,15 +6,20 @@
 import edu.cornell.mannlib.vitro.webapp.edit.n3editing.VTwo.ProcessRdfForm;
 import edu.cornell.mannlib.vitro.webapp.modelaccess.ModelAccess;
 import edu.cornell.mannlib.vitro.webapp.utils.configuration.Property;
+import org.apache.commons.lang3.StringUtils;
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
 import org.apache.jena.rdf.model.Literal;
 import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.ResourceFactory;
 
 import java.util.*;
+import java.util.stream.Collectors;
 
 public class N3Template extends Operation implements Template {
 
+	private static final String ANY_URI="http://www.w3.org/2001/XMLSchema#anyURI";
```

Comment:
I believe we have a constant for ANY_URI defined a couple of times, probably we should change access modifier (to public), and defined that at one place.

Reply:
Just looked at it, and it seams we have this constant only in N3Templates and Parameters. Again, once we decide where variable substitution belongs, we can leave this constant either here, or inside Parameters class